src/workflows/pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "airqo_etl_utils"
+version = "1.0.0"
+description = "AirQo Airflow custom python package"
+authors = ["AirQo <data@airqo.net>"]
+license = "MIT"
+keywords = ["python", "airflow", "AirQo"]
+include = ["**/*.json"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+apache-airflow = "*"
```

Comment:
Hi @MartinKalema 
Don't you think it would be appropriate to specify exact versions, not to face issues caused by updates that might later introduce breaking changes 

Or having depandence constraints like `apache-airflow = ">=2.0.0,<3.0.0"` instead of `apache-airflow = "*"`

Reply:
Thanks @MartinKalema for the PR. I'm learning from you at the same time